### ISSUE TITLE
USB-C voltage trigger

### ISSUE THREAD

**Issue by pepsi-zero** [Xinyuan-LilyGO/LilyGo-T-OI-PLUS#15]
CC1 is connected to CC2 and share only 1 resistor. According to the USB PD2 design documentation must have CC1 and CC2 each have it’s own 5k1 resistor.

**Comment by LeDominik** [comment]
Yeah, this is probably the reason why this board is super-sensitive with chargers... The initial Raspberry Pi 4 had that "error" as well...
